app/main/menu/submenu-items.js

```diff
@@ -140,6 +140,9 @@ items.reloadApp = {
   click(item, focusedWindow) {
     if (focusedWindow) {
       focusedWindow.reload();
+      focusedWindow.webContents.once('did-finish-load', () => {
```

Comment:
Is this one right?  I don't expect to get a new tab when I reload.  Also, does "Reload App" reload _all_ open `BrowserWindows`?  If not, why not?  Should we re-label this to 'Reload Window' or similar?